Docs/Configuration.tex

```diff
@@ -1404,10 +1404,10 @@ \subsection{Quirks Properties}\label{booterpropsquirks}
     \item KASLR (slide) is unsupported (this is macOS 10.7 or older)
   \end{itemize}
 
-  This quirk requires \texttt{ProvideCustomSlide} to also be enabled
-  and typically needs \texttt{AvoidRuntimeDefrag} to be set for this to
-  work correctly. Hibernation is not supported when booting with a relocation
-  block (but relocation block is not always used when the quirk is enabled).
+  This quirk requires \texttt{ProvideCustomSlide} to be enabled
+  and typically requires \texttt{AvoidRuntimeDefrag} to be set to work
```

Comment:
`and typically requires \texttt{AvoidRuntimeDefrag} to be enabled as well for correct functioning.`

Reply:
Changed to the same intent